ledger/ledger-api-bench-tool/src/main/scala/com/daml/ledger/api/benchtool/submission/CommandSubmitter.scala

```diff
@@ -225,24 +226,28 @@ case class CommandSubmitter(
             .map(cmds => cmds.head._1 -> cmds.map(_._2).toList)
             .buffer(maxInFlightCommands, OverflowStrategy.backpressure)
             .mapAsync(maxInFlightCommands) { case (index, commands) =>
-              timed(submitLatencyTimer, metricsManager)(
+              timed(submitLatencyTimer, metricsManager) {
                 submit(
                   id = names.commandId(index),
                   actAs = baseActAs ++ generator.nextExtraCommandSubmitters(),
                   commands = commands.flatten,
                   applicationId = generator.nextApplicationId(),
                   useSubmitAndWait = config.waitForSubmission,
                 )
-              )
+              }
                 .map(_ => index + commands.length - 1)
-                .recoverWith { case ex =>
-                  Future.failed {
+                .recoverWith {
+                  case e: io.grpc.StatusRuntimeException
+                      if e.getStatus.getCode == Status.Code.ABORTED =>
```

Comment:
Not all `ABORTED` mean backoff, right?
If so, it'd better to use a dedicated error code and match it precisely using `com.daml.error.utils.ErrorDetails#matches`

Reply:
For DA all `ABORTED` do mean backoff.  This code was chosen because the Netty infrastructure do not use this code.